src/components/Form/Radio.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react'
 import { css } from 'glamor'
-import { fontFamilies } from '../../theme/fonts'
+import { fontStyles } from '../../theme/fonts'
+import { pxToRem } from '../Typography/utils'
 import { useColorContext } from '../Colors/useColorContext'
 
 const styles = {
   label: css({
-    fontSize: 16,
-    lineHeight: '20px',
-    fontFamily: fontFamilies.sansSerifRegular,
     cursor: 'pointer'
   }),
+  withText: css({
+    ...fontStyles.sansSerifRegular,
```

Comment:
Whats the benefit of using this with pxToRem instead of just ...fontStyles.sansSerifRegular16 ?